components/Repositories/RossChart.tsx

```diff
@@ -108,6 +113,13 @@ export default function RossChart({ stats, isLoading, error, range, className }:
                 <h3 className="text-xs xl:text-sm text-slate-500">Average per week</h3>
                 <p className="font-semibold text-xl xl:text-3xl">{humanizeNumber(rangedAverage)}</p>
               </div>
+              <div>
+                <div className="flex gap-2 items-center">
+                  <h3 className="text-xs xl:text-sm text-slate-500">Outside Contributors</h3>
```

Comment:
no need to make this a [heading](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements). Headings are typically for breaking up content/Table of Contents. If it's for styling, opt for another tag.

Reply:
It's to match the other h3 tags throughout the page as the label I've used for the stats in the different charts. Would you recommend I use just another p tag?

Reply:
Another tag would probably make more sense, but if it's everywhere else, for now, leave it and we can do a follow up PR.